<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>562</width>
         <height>353</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>857</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actAddFolder"/>
   <addaction name="actAddFiles"/>
   <addaction name="actDeleteItem"/>
   <addaction name="actScanItems"/>
   <addaction name="actZoomIn"/>
   <addaction name="actZoomOut"/>
   <addaction name="actZoomRealSize"/>
   <addaction name="actZoomFitW"/>
   <addaction name="actZoomFitH"/>
   <addaction name="actDockFloat"/>
   <addaction name="actDockVisible"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string notr="true">节点</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>节点类型</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actAddFolder">
   <property name="icon">
    <iconset resource="icom.qrc">
     <normaloff>:/icons/imgs/添加目录.png</normaloff>:/icons/imgs/添加目录.png</iconset>
   </property>
   <property name="text">
    <string>添加目录</string>
   </property>
   <property name="toolTip">
    <string>添加目录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actAddFiles">
   <property name="icon">
    <iconset resource="icom.qrc">
     <normaloff>:/icons/imgs/添加文件.png</normaloff>:/icons/imgs/添加文件.png</iconset>
   </property>
   <property name="text">
    <string>添加文本</string>
   </property>
   <property name="toolTip">
    <string>添加文本</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actZoomIn">
   <property name="icon">
    <iconset resource="icom.qrc">
     <normaloff>:/icons/imgs/放大.png</normaloff>:/icons/imgs/放大.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actZoomOut">
   <property name="icon">
    <iconset resource="icom.qrc">
     <normaloff>:/icons/imgs/缩小.png</normaloff>:/icons/imgs/缩小.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actZoomRealSize">
   <property name="icon">
    <iconset resource="icom.qrc">
     <normaloff>:/icons/imgs/实际大小-经过.png</normaloff>:/icons/imgs/实际大小-经过.png</iconset>
   </property>
   <property name="text">
    <string>实际大小</string>
   </property>
   <property name="toolTip">
    <string>实际大小</string>
   </property>
  </action>
  <action name="actZoomFitW">
   <property name="icon">
    <iconset resource="icom.qrc">
     <normaloff>:/icons/imgs/适合宽度.png</normaloff>:/icons/imgs/适合宽度.png</iconset>
   </property>
   <property name="text">
    <string>适合宽度</string>
   </property>
   <property name="toolTip">
    <string>适合宽度</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actDeleteItem">
   <property name="icon">
    <iconset resource="icom.qrc">
     <normaloff>:/icons/imgs/删除节点.png</normaloff>:/icons/imgs/删除节点.png</iconset>
   </property>
   <property name="text">
    <string>删除节点</string>
   </property>
   <property name="toolTip">
    <string>删除节点</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="icom.qrc">
     <normaloff>:/icons/imgs/退出(2).png</normaloff>:/icons/imgs/退出(2).png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actZoomFitH">
   <property name="icon">
    <iconset resource="icom.qrc">
     <normaloff>:/icons/imgs/高度.png</normaloff>:/icons/imgs/高度.png</iconset>
   </property>
   <property name="text">
    <string>适合高度</string>
   </property>
   <property name="toolTip">
    <string>适合高度</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actScanItems">
   <property name="icon">
    <iconset resource="icom.qrc">
     <normaloff>:/icons/imgs/类型-遍历节点.png</normaloff>:/icons/imgs/类型-遍历节点.png</iconset>
   </property>
   <property name="text">
    <string>遍历节点</string>
   </property>
   <property name="toolTip">
    <string>遍历节点</string>
   </property>
  </action>
  <action name="actDockVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icom.qrc">
     <normaloff>:/icons/imgs/可见.png</normaloff>:/icons/imgs/可见.png</iconset>
   </property>
   <property name="text">
    <string>窗体可见</string>
   </property>
   <property name="toolTip">
    <string>窗体可见</string>
   </property>
  </action>
  <action name="actDockFloat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icom.qrc">
     <normaloff>:/icons/imgs/浮动窗口.png</normaloff>:/icons/imgs/浮动窗口.png</iconset>
   </property>
   <property name="text">
    <string>窗体浮动</string>
   </property>
   <property name="toolTip">
    <string>窗体浮动</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icom.qrc"/>
 </resources>
 <connections/>
</ui>
